cookbooks/bcpc/recipes/zabbix-head.rb

```diff
@@ -20,7 +20,8 @@
 include_recipe "bcpc::mysql"
 include_recipe "bcpc::apache2"
 
-make_config('mysql-zabbix-user', "zabbix")
+mysql_zabbix_user = 'zabbix'
```

Comment:
I would not recommend directly setting this variable. First, `make_config()` the user and `get_config!()` the user anywhere you need the string; you could cache the result of `get_config!()` in `mysql_zabbix_user` though.